### PR TITLE
Meta/2025 meta 5

### PR DESCRIPTION
**Pull request by sorja**
Part of #1502

Section 5
- [x]  5a (@sorja )
- [x]  5b (@sorja )
- [x]  5c(@sorja ) no changes

spec:
Screenshot 2022-10-25 at 11 27 18

2020 5a:

Screenshot 2022-10-25 at 11 28 16

2025 5a:

Screenshot 2022-10-25 at 11 28 35

2020 5b:

Screenshot 2022-10-25 at 11 28 54

2025 5b:

Screenshot 2022-10-25 at 11 29 14

2020 5c: no change
2025 5c: no change

### CODE REVIEW

**Review by rfontanarosa — APPROVED** [review]

**Review by minotogna — CHANGES_REQUESTED** [review]
checke Fra 2025 categories tittle to all tables

File: src/i18n/resources/en/fra.js [hunk 1]
```diff
@@ -7,6 +7,16 @@ module.exports = {
   forestArea100HaYear: 'Forest area (1000 ha)',
 
   // tables
+  areaAffectedByFire: {
+    areaUnitLabel2025: 'Area affected (1000 ha)',
+  },
+
+  disturbances: {
+    disturbances2025: 'Forest damage',
```
Comment (minotogna) [inline src/i18n/resources/en/fra.js]: can this be forestDamage ?

File: src/i18n/resources/en/fra.js [hunk 2]
```diff
@@ -7,6 +7,16 @@ module.exports = {
   forestArea100HaYear: 'Forest area (1000 ha)',
 
   // tables
+  areaAffectedByFire: {
+    areaUnitLabel2025: 'Area affected (1000 ha)',
```
Comment (minotogna) [inline src/i18n/resources/en/fra.js]: does this match 2025  data export unit?

File: src/i18n/resources/en/fra.js [hunk 3]
```diff
@@ -7,6 +7,16 @@ module.exports = {
...
+  },
+
+  disturbances: {
+    disturbances2025: 'Forest damage',
+    categoryHeader: 'Predominant cause',
+    areaUnitLabel2025: 'Forest area affected (1000 ha)',
```
Comment (minotogna) [inline src/i18n/resources/en/fra.js]: can these be predominantCause and forestAreaAffected ? does it match 2025 data export unit label?

File: src/i18n/resources/en/fra.js (hunk 2 above)
Comment (minotogna) [reply src/i18n/resources/en/fra.js]: can this be areaAffected ?

File: src/i18n/resources/en/fra.js (hunk 1 above)
Comment (sorja) [reply src/i18n/resources/en/fra.js]: Definitely, at first I named it 'forestDamage' but renamed it to keep track of what has changed from previous version

I will change this

**Comment by sorja** [comment]
Good catch on the dataExport label - totally missed migrating this 👍

**Comment by sorja** [comment]
> checke Fra 2025 categories tittle to all tables

its correct

**Comment by sorja** [comment]
DataExport unit labels come from fra variables, so they should be in place

Here is video to verify:

https://user-images.githubusercontent.com/5508251/197974957-96d2ef67-b0fc-49b2-b2c7-74e9aa21b6c0.mov

**Review by minotogna — APPROVED** [review]

**Pull request merged**